consensus/istanbul/core/roundchange.go

```diff
@@ -122,21 +123,24 @@ func (c *core) handleRoundChange(msg *message, src istanbul.Validator) error {
 
 	if num == numStartNewRound && (c.waitingForRoundChange || cv.Round.Cmp(roundView.Round) < 0) {
 		// We've received enough ROUND CHANGE messages, start a new round immediately.
-		logger.Warn("[RC] Received 2f+1 Round Change Messages. Starting new round")
+		logger.Warn("[RC] Received 2f+1 Round Change Messages. Starting new round",
+			"currentRound", cv.Round.Uint64(), "newRound", roundView.Round.Uint64())
 		c.startNewRound(roundView.Round)
 		return nil
 	} else if c.waitingForRoundChange && num == numCatchUp {
 		// Once we received enough ROUND CHANGE messages, those messages form a weak certificate.
 		// If our round number is smaller than the certificate's round number, we would
 		// try to catch up the round number.
 		if cv.Round.Cmp(roundView.Round) < 0 {
-			logger.Warn("[RC] Send round change because we have F+1 roundchange messages")
+			logger.Warn("[RC] Send round change because we have F+1 round change messages",
+				"currentRound", cv.Round.Uint64(), "msgRound", roundView.Round.Uint64())
 			c.sendRoundChange(roundView.Round)
 		}
 		return nil
 	} else if cv.Round.Cmp(roundView.Round) < 0 {
 		// Only gossip the message with current round to other validators.
-		logger.Warn("[RC] Received round is bigger but not enough number of messages. Message ignored")
+		logger.Warn("[RC] Received round is bigger but not enough number of messages. Message ignored",
```

Comment:
The round change messages are stored and the log `Message ignored` seems confusing. Also, `Warn` looks like it is pretty high because a round change can happen frequently when in a network disorder.
```suggestion
		logger.Info("[RC] Received round is bigger but not enough number of messages. Nothing happens",
```

Reply:
Thanks. I changed them